internal/node/node.go

```diff
@@ -149,6 +149,8 @@ func NewBus(cfg BusConfig, dir string, seed types.PrivateKey, l *zap.Logger) (ht
 		Logger:                        l.Sugar(),
 		GormLogger:                    dbLogger,
 		RetryTransactionIntervals:     []time.Duration{200 * time.Millisecond, 500 * time.Millisecond, time.Second, 3 * time.Second, 10 * time.Second, 10 * time.Second},
+		LongQueryDuration:             cfg.DatabaseLog.SlowThreshold,
+		LongTxDuration:                cfg.DatabaseLog.SlowThreshold,
```

Comment:
Same as the logged types this was taken from `hostd`. I like the idea of logging slow txns because right now we don't seem to have that. If a txn consists of 1000 fast queries we might not log them even though the txn as a whole was slow.

Since I don't have any good datapoints yet I set it to the same value as the slow query duration for now.